### ISSUE TITLE
TOSTpaired.raw significance discrepancy

### ISSUE THREAD

**Issue by ontogenerator** [Lakens/TOSTER#25]
Dear developers,

I am currently making a shiny App for power analysis and experimental design assistance, based primarily on your TOSTER (v0.3) R package. When working on examples with repeated measures I ran into an inconsistency between the graphical output of the TOSTpaired.raw function and the significance of the reported TOST results.

For instance, in this example:

```
TOSTpaired.raw(n = 10, m1 = 24, m2 = 22, sd1 = 2, sd2 = 2, r12 = 0.5,
               low_eqbound = -3, high_eqbound = 3, alpha = 0.05)
```

the 90% CI overlaps with the SESOI boundary, yet the resulting p values are both significant, with the overall result of the TOST being significant (i. e. equivalent).

After looking deeper into the code, I may have found the reason for this behavior.

The current version of the code for the t value calculation is (e.g. for t1):

`t1 <- ((m1_s - m2_s) - (low_eqbound * sdif))/se`

However, according to my interpretation of section 1.3 in Mara & Cribbie (2012), the formula should be:

`t1 <- ((m1_s - m2_s) - low_eqbound)/se`

Of course, I may have overlooked something, but it would be nice to know either way.

Cheers,

Vladislav

**Comment by Lakens** [comment]
Hi, I'm traveling but it looks like we did not remove the multiplication by sdiff when creating to raw function from the standardized function. Thanks for finding this! We will update it. Can you email me with info about the shiny app? Thanks!

________________________________

**Comment by Lakens** [comment]
Hi, I fixed the code, and submitted a new version (3.1) to CRAN. The version on GitHub is corrected. Thanks for informing me - you rock!
